kolibri/plugins/facility/assets/src/views/users/NewUsersPage.vue

```diff
@@ -357,7 +369,26 @@
         fetchClasses();
       });
 
+      // Alerting users about enrollment/assignment errors
+      const alertDismissed = ref(false);
+
+      const showingInvalidUsers = computed(
+        () => !alertDismissed.value && Boolean(route.query.by_ids),
+      );
```

Comment:
I am not seeing any other assignment to the `asertDismissed` ref, so this is always `false` in the entire component life cycle? Should we also check the `failedActionType` ref here?